src/redux/categories/categories.js

```diff
@@ -0,0 +1,28 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const CHECKSTATUS = 'bookstore/categories/CHECKSTATUS'; // name
+
+const initialState = []; // initial state
+
+// reducers
+const categoryReducer = (state = initialState, action = {}) => {
+  switch (action.type) {
+    case CHECKSTATUS:
+      return [
+        ...state, {
+          id: uuidv4(),
+          text: action.status.text,
```

Comment:
**OPTIONAL** - Since this reducer is only meant to return a text, I think it is not necessary to have an id, it is easier and cleaner to just return the required text. i.e `return 'Under Construction'`